src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,72 @@
+package duke.parser;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.exception.DukeWrongInputFormatException;
+
+/**
+ * Utility Class to parse the command given by the user.
+ */
+public class Parser {
+
+    /**
+     * Parse the command given by the user.
```

Comment:
Perhaps it should be parses for the method header comment.

Reply:
Thanks for catching this!